src/test/1.settings.test.ts

```diff
@@ -60,4 +60,25 @@ describe("Extension settings tests", () => {
     const enablement = await setting.getValue();
     expect(enablement).to.equal(true);
   });
+
+  it("verify vorpal checkbox exists in the settings", async function () {
+    settingsEditor = await bench.openSettings();
+    const vorpalCheckbox = await settingsEditor.findSetting(
+      "ActivateVorpalAutoScanning",
+      "Checkmarx Vorpal"
```

Comment:
maybe make "ActivateVorpalAutoScanning", "Checkmarx Vorpal" as constants? I saw that those values are also being use in other places